bugtool/cmd/configuration.go

```diff
@@ -419,6 +419,9 @@ func copyCiliumInfoCommands(cmdDir string, k8sPods []string) []string {
 			kubectlArg := fmt.Sprintf("%s/%s:%s", k8sNamespace, pod, stateDir)
 			// kubectl cp kube-system/cilium-xrzwr:/var/run/cilium/state cilium-xrzwr-state
 			commands = append(commands, fmt.Sprintf("kubectl cp %s %s", kubectlArg, dst))
+			if excludeObjectFiles {
+				commands = append(commands, fmt.Sprintf("rm %s/*.o", dst))
```

Comment:
How did you test this? I wouldn't expect it to work given the object files are not located at the root of the state directory.

Reply:
@pchaigno So, should I use command `find dst -name "*.o" -type f -print0 | xargs -0 rm -f` instead.

Reply:
I think that's overkill; the object files are in a handful of subdirectories IIRC. We should be able to reach them with some basic wildcard pattern(s).

Reply:
@pchaigno So, could you please suggest what command should be used here instead.

Reply:
I don't know off the top of my head. You can probably check that by deploying Cilium and looking at where the object files are in the state directory inside a Cilium agent container (`/var/run/cilium/state/`).